includes/admin/class-llms-sendwp.php

```diff
@@ -43,10 +44,11 @@ class LLMS_SendWP extends LLMS_Abstract_Email_Provider {
 	 * @since 3.36.1
 	 * @since 3.37.0 Sanitize URLS returned by SendWP functions and add nonce verification.
 	 * @deprecated 3.40.0 Method to be made protected and should not be called publicly.
+	 * @since [version] Changed access from public to protected.
 	 *
 	 * @return array
 	 */
-	public function do_remote_install() { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found -- Intentional for backwards compat.
+	protected function do_remote_install() {
```

Comment:
The parent method should already be protected so this method can just be removed from this class 

Double check me on that